rust/src/tests/rmdir.rs

```diff
@@ -141,3 +150,33 @@ crate::test_case! {
 fn einval_dot(ctx: &mut TestContext) {
     assert_eq!(rmdir(&ctx.base_path().join(".")), Err(Errno::EINVAL));
 }
+
+crate::test_case! {
+    /// rmdir returns EEXIST or ENOTEMPTY if the last component of the path is '..'
+    // rmdir/12.t
+    eexist_enotempty_dotdot
+}
+fn eexist_enotempty_dotdot(ctx: &mut TestContext) {
+    // TODO: Not conforming to POSIX on FreeBSD
+    // According to POSIX: EEXIST or ENOTEMPTY:
+    // The path argument names a directory that is
+    // not an empty directory,
+    // or there are hard links to the directory other than dot or a single entry in dot-dot.
+    #[cfg(not(target_os = "freebsd"))]
```

Comment:
Ideally we'd skip this test, rather than not compile it.  But there isn't yet a facility to do that.